manifests/a/AmazonWebServices/AmazonEC2Launch/2.0.2046.0/AmazonWebServices.AmazonEC2Launch.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: AmazonWebServices.AmazonEC2Launch
+PackageVersion: 2.0.2046.0
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{2C83974C-ADA4-4388-8180-CB2EBFAE624F}'
+AppsAndFeaturesEntries:
+- DisplayName: Amazon EC2Launch
+  Publisher: Amazon Web Services
+  ProductCode: '{2C83974C-ADA4-4388-8180-CB2EBFAE624F}'
+  UpgradeCode: '{7D032E02-C708-4C10-B811-5F1ACFD673DB}'

```

```suggestion
AppsAndFeaturesEntries:
- UpgradeCode: '{7D032E02-C708-4C10-B811-5F1ACFD673DB}'
```

Most of the ARP fields aren't needed since they're an exact match with other values already present in the manifest